index.html

```diff
@@ -63,7 +63,212 @@ <h1 class="header">I'm Lawrence Glad to see you</h1>
 
         </section>
 
-    </div>
+        <div class="contained">
```

Comment:
- [ ] Kindly wrap your work section in a `section` tag for accessibility and semantic reasons
- [ ] Kindly align the content in your `see project` button to the center to match the figma design
 ### Your design

![image](https://user-images.githubusercontent.com/98453979/192954880-3b51d277-52a4-4b58-8151-94c5c00708bf.png)
### checkout figma

![image](https://user-images.githubusercontent.com/98453979/192955048-f8545342-1ea0-4b07-9a4c-c69bfc05db56.png)

- [ ] Kindly adjust the spacing of elements on your project cards to match the Figma design

![image](https://user-images.githubusercontent.com/98453979/192955749-a0f847e7-916d-4b86-a20d-2e595eeaa35a.png)

#### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/192955853-fafd073a-4f9c-4e85-b1b4-787ea5804379.png)